lib/src/markdown.component.ts

```diff
@@ -1,27 +1,40 @@
+import { CommonModule } from '@angular/common';
 import {
   AfterViewInit,
   Component,
   ElementRef,
   EventEmitter,
+  HostListener,
   Input,
   OnChanges,
   OnDestroy,
+  Optional,
   Output,
   TemplateRef,
   Type,
   ViewContainerRef,
 } from '@angular/core';
-import { Subject } from 'rxjs';
-import { takeUntil } from 'rxjs/operators';
+import { NavigationExtras, Router } from '@angular/router';
+import { from, merge, Subject } from 'rxjs';
+import { filter, map, switchMap, takeUntil, tap } from 'rxjs/operators';
 import { KatexOptions } from './katex-options';
 import { MarkdownService, ParseOptions, RenderOptions } from './markdown.service';
 import { MermaidAPI } from './mermaid-options';
 import { PrismPlugin } from './prism-plugin';
 
+export interface MarkdownRouterLinkOptions {
+  global?: NavigationExtras;
+  paths?: { [path: string]: NavigationExtras | undefined };
+}
+
 @Component({
   // eslint-disable-next-line @angular-eslint/component-selector
   selector: 'markdown, [markdown]',
-  template: '<ng-content></ng-content>',
+  template: `
+  <ng-container *ngIf="changed$ | async">
+  <ng-content></ng-content>
```

Comment:
Isn't the closing `ng-container` missing?

Reply:
Yeah, I just fixed it.